configs/serviceops/actions.yml

```diff
@@ -0,0 +1,4 @@
+actions:
+- handler: versionList
+  instances:
+  - MyListConstructor
```

Comment:
This name is confusing.
We are **not** constructing a `list` here. 
We are constructing `handlerRequestArgs`